src/common/context.rs

```diff
@@ -98,6 +91,7 @@ impl Context {
             stdin: sudo_options.stdin,
             non_interactive: sudo_options.non_interactive,
             process: Process::new(),
+            auth_user: AuthUser::resolve_root_for_rootpw()?,
```

Comment:
Is this value ever used without getting overwritten other than in `test_build_context`?

Reply:
I was asking myself the same question but somehow didn't remove it. The `Context` object is something I'd like to simplify/split up in the future anyway (or maybe you may feel inclined to do so).

Reply:
See #972 for a resolution, or we can decide to more thoroughly refactor the `AuthPlugin` pipeline in a different PR (and then following that, maybe have a stern look at the `Context` object). I do concur that AuthUser doesn't really belong here.

Reply:
We may indeed want to make more thorough improvements in the future, but for now #972 works fine.